ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/methods/EngineExchangeTransitionConfigurationV1.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.ethereum.executionclient.methods;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import tech.pegasys.teku.ethereum.executionclient.ExecutionEngineClient;
+import tech.pegasys.teku.ethereum.executionclient.response.ResponseUnwrapper;
+import tech.pegasys.teku.ethereum.executionclient.schema.TransitionConfigurationV1;
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
+import tech.pegasys.teku.spec.executionlayer.TransitionConfiguration;
+
+public class EngineExchangeTransitionConfigurationV1
+    extends AbstractEngineJsonRpcMethod<TransitionConfiguration> {
+
+  private static final Logger LOG = LogManager.getLogger();
+
+  public EngineExchangeTransitionConfigurationV1(
+      final ExecutionEngineClient executionEngineClient) {
+    super(executionEngineClient);
+  }
+
+  @Override
+  public String getName() {
+    return "engine_exchangeTransitionConfiguration";
+  }
+
+  @Override
+  public int getVersion() {
+    return 1;
+  }
+
+  @Override
+  public SafeFuture<TransitionConfiguration> execute(final JsonRpcRequestParams params) {
+    final TransitionConfiguration transitionConfiguration =
+        params.getRequiredParameter(0, TransitionConfiguration.class);
+
+    LOG.trace(
+        "calling engineExchangeTransitionConfiguration(transitionConfiguration={})",
```

Comment:
i wonder if its worth in all of these trace fns having them reference the versioned name function rather than hard coding the name in each one...
```
"calling {}(transitionConfiguration={})",
getVersionedName(),
transitionConfiguration
```

Reply:
Yeah, I can do that for all methods at once in a separate change.